Bio/Phylo/PhyloXML.py

```diff
@@ -1384,8 +1429,7 @@ def __init__(self, value, desc=None, type=None):
         self.type = type
 
     def __str__(self):
+        """Return string representation of Uri."""
         if self.value:
             return self.value
-        """Initialize the class."""
-        """Initialize the class."""
```

Comment:
Those look like an accidental paste - well spotted. I wonder if there is a flake8 setting to report this kind of thing?